### PR TITLE
[Motions 2022 11 lwg 8] P2704R0 C++ Standard Library Immediate Issues

### LINKED ISSUE #5969: [2022-11 LWG Motion 8] P2704R0 C++ Standard Library Immediate Issues
https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2704r0.html

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #5969.
Fixes cplusplus/papers#1393

### CODE REVIEW

File: source/xrefdelta.tex [hunk 1]
```diff
@@ -106,5 +106,9 @@
 % LWG3659 Consider ATOMIC_FLAG_INIT undeprecation
 \removedxref{depr.atomics.flag}
 
+% LWG3818 Exposition-only concepts are not described in library intro
+\movedxref{expos.only.func}{expos.only.entity}
+\removedxref{expos.only.types}
```
Comment (JohelEGP) [inline source/xrefdelta.tex:111]: Why not `\movedxref`? It is removed, but the specification was merged to [expos.only.entity].

**Review by JohelEGP — CHANGES_REQUESTED** [review]
LWG3817 LWG3818 LWG3826 are not part of P2704R0. I reviewed them nonetheless.

File: source/lib-intro.tex [hunk 2]
```diff
@@ -478,13 +478,14 @@
...
-\rSec3[expos.only.func]{Exposition-only functions}
+\rSec3[expos.only.entity]{Exposition-only entities, etc.}
 
 \pnum
-Several function templates
+Several entities and \grammarterm{typedef-name}{s}
 defined in \ref{\firstlibchapter} through \ref{\lastlibchapter} and \ref{depr}
 are only defined for the purpose of exposition.
-The declaration of such a function is followed by a comment ending in \expos.
+The declaration of such an entity and \grammarterm{typedef-name}
```
Comment (timsong-cpp) [inline source/lib-intro.tex:487]:
```suggestion
The declaration of such an entity or \grammarterm{typedef-name}
```

**Comment by timsong-cpp** [comment]
> [LWG3817](https://wg21.link/lwg3817) [LWG3818](https://wg21.link/lwg3818) [LWG3826](https://wg21.link/lwg3826) are not part of P2704R0. 

They are in the version attached to the wiki. @jwakely - I guess the paper tracking system need to be updated?

**Comment by jwakely** [comment]
Oops yes, I didn't put the latest version on isocpp.org

**Comment by jwakely** [comment]
The document on isocpp.org has been updated to match what we voted on in Kona.

**Comment by burblebee** [comment]
> [LWG3817](https://wg21.link/lwg3817) [LWG3818](https://wg21.link/lwg3818) [LWG3826](https://wg21.link/lwg3826) are not part of P2704R0. I reviewed them nonetheless.

What paper are you looking at? They're in the document I'm looking at from the straw polls (https://wiki.edg.com/pub/Wg21kona2022/StrawPolls/p2704r0.html?twiki_redirect_cache=8ee3b437fe2dbe928dd162d653e0e49a).

**Comment by JohelEGP** [comment]
I used https://wg21.link/P2704R0.

File: source/xrefdelta.tex (hunk 1 above)
Comment (burblebee) [reply source/xrefdelta.tex:111]: The section itself (name and number) no longer exist, hence `\removedxref`. `\movedxref` is for sections that are renamed. We don't have a way of documenting that a section contents were merged into another section afaik, we just move the contents.

File: source/lib-intro.tex (hunk 2 above)
Comment (burblebee) [reply source/lib-intro.tex:487]: This is from LWG3818 which supposedly was not intended to be part of the paper.

Comment (JohelEGP) [reply source/lib-intro.tex:487]: I think it was intended to be. See ~3 comments starting from https://github.com/cplusplus/draft/pull/5982#issuecomment-1320991610.

File: source/xrefdelta.tex (hunk 1 above)
Comment (burblebee) [reply source/xrefdelta.tex:111]: FYI, this is from LWG3818 which was not intended to be part of the paper.

**Comment by burblebee** [comment]
What's supposed to be the "official" paper that we use for the motions? I'm using the ones linked in the strawpolls themselves since my understanding is that *that* is what was voted on. If that paper was out of date, then the people that voted on the older version it may object to having things change out from underneath them.

@tkoeppe what to do here? Should I remove [LWG3817](https://wg21.link/lwg3817) [LWG3818](https://wg21.link/lwg3818) [LWG3826](https://wg21.link/lwg3826) and force-push?

**Comment by burblebee** [comment]
Ok... I think folks are saying that the copy in the strawpolls *is* the correct one, right? So [LWG3817](https://wg21.link/lwg3817) [LWG3818](https://wg21.link/lwg3818) [LWG3826](https://wg21.link/lwg3826) are supposed to be in.

**Comment by jwakely** [comment]
> Ok... I think folks are saying that the copy in the strawpolls _is_ the correct one, right? So [LWG3817](https://wg21.link/lwg3817) [LWG3818](https://wg21.link/lwg3818) [LWG3826](https://wg21.link/lwg3826) are supposed to be in.

Yes. They're in https://wg21.link/P2704R0 now anyway, it's been updated to match the one we voted on.

**Review by JohelEGP — APPROVED** [review]

**Pull request merged**